sympy/calculus/util.py

```diff
@@ -1287,27 +1287,44 @@ def __rtruediv__(self, other):
             if other.is_extended_real:
                 if other.is_zero:
                     return S.Zero
-                if (self.min.is_extended_nonpositive and self.max.is_extended_nonnegative):
+                if (self.min.is_extended_nonpositive and
+                        self.max.is_extended_nonnegative):
                     if self.min.is_zero:
+                        invM = 1/self.max if not self.max.is_infinite else 0
                         if other.is_extended_positive:
-                            return AccumBounds(Mul(other, 1 / self.max), oo)
+                            return AccumBounds(Mul(other, invM), oo)
                         if other.is_extended_negative:
-                            return AccumBounds(-oo, Mul(other, 1 / self.max))
+                            return AccumBounds(-oo, Mul(other, invM))
                     if self.max.is_zero:
+                        invM = 1/self.min if not self.min.is_infinite else 0
                         if other.is_extended_positive:
-                            return AccumBounds(-oo, Mul(other, 1 / self.min))
+                            return AccumBounds(-oo, Mul(other, invM))
                         if other.is_extended_negative:
-                            return AccumBounds(Mul(other, 1 / self.min), oo)
+                            return AccumBounds(Mul(other, invM), oo)
                     return AccumBounds(-oo, oo)
                 else:
-                    return AccumBounds(Min(other / self.min, other / self.max),
-                                       Max(other / self.min, other / self.max))
+                    args = [other/i if not i.is_infinite
+                        else 0 for i in self.args]
+                    if args[0] > args[1]:
```

Comment:
I didn't realize this line from the test failure was actually added in this PR. We can't assume args has non-symbolic values here.